go/worker/helpers.go

```diff
@@ -40,22 +39,22 @@ func initOLBaseDir(baseDir string, dockerBaseImage string) error {
 
 	// PART 2: various files/dirs on top of the extracted image
 	fmt.Printf("\tCreate handler/host/packages/resolve.conf over base image.\n")
-	if err := os.Mkdir(path.Join(baseDir, "handler"), 0700); err != nil {
+	if err := os.Mkdir(path.Join(baseDir, "handler"), 0755); err != nil {
```

Comment:
Why do we need to change all these permissions?  If they do everything (init and up) as root, it is fine, right?

And if they do init with sudo, and up with ID mapping, wouldn't everything be created by user 0 and used by user 0?  Not sure we want to make all these things visible to every user running on Linux.